pkg/server/httpAPI.go

```diff
@@ -466,6 +476,32 @@ func (h *HTTPHandler) serveConfig(rw http.ResponseWriter, req *http.Request) {
 	h.Logger.Info("Served config for ETCD instance.")
 }
 
+// GetClusterState returns the Cluster state either `new` or `existing`.
+func (h *HTTPHandler) GetClusterState(ctx context.Context, clusterSize int, clientSet client.Client, podName string, podNS string) string {
+	if clusterSize == 1 {
```

Comment:
Shouldn't this case be handled within `GetInitialClusterState` method?

Reply:
I have re-name `GetInitialClusterState` the func to avoid ambiguity.